tests/asgi/test_example_asgi.py

```diff
@@ -1,223 +0,0 @@
-# examples/things_advanced_asgi.py
```

Comment:
How come it was removed?

Reply:
It is no longer needed. This is just a copy from the examples folder.
These are already imported via `importlib` and tested in `tests/test_examples.py`.

Reply:
Ok!

Reply:
For reference, this has been improved recently in [#2261](https://github.com/falconry/falcon/pull/2261/files#diff-ef88d18dff1f7f3977c40a0649ba44b2c41d89676352a4d5c1fd4f3e7c81dfc7), I just forgot to remove this file in that PR. I did remove the WSGI counterpart.